src/App.tsx

```diff
@@ -64,7 +68,11 @@ const App = () => {
                     <AuthProvider>
                         <Route exact path="/signin" component={SignIn} />
                         <Route exact path="/signout" component={SignIn} />
-                        <PrivateRoute exact path="/risk" component={Risk} />
```

Comment:
can we keep /risk and redirect to ethereum default url? i think /risk rn doesnt load and strategist have the original bookmark